tests/conftest.py

```diff
@@ -197,3 +198,69 @@ def roc_multi_classification_values_set2():
     y_score = classifier.fit(X_train, y_train).predict_proba(X_test)
 
     return y_test, y_score
+
+
+@pytest.fixture(scope="session")
+def precision_recall_binary_classification():
+    data = datasets.make_classification(
+        n_samples=50,
+        n_features=10,
+        n_informative=5,
+        n_classes=2,
+        class_sep=0.65,
+        random_state=0,
+    )
+    X = data[0]
+    y = data[1]
+    X_train, X_test, y_train, y_true = train_test_split(
+        X, y, test_size=0.2, random_state=0
+    )
+
+    est = RandomForestClassifier(random_state=0)
+    est.fit(X_train, y_train)
+    y_score = est.predict_proba(X_test)
+    return y_true, y_score
+
+
+@pytest.fixture(scope="session")
+def precision_recall_multiclass_classification():
+    data = datasets.make_classification(
+        n_samples=50,
+        n_features=10,
+        n_informative=5,
+        n_classes=3,
+        class_sep=0.65,
+        random_state=0,
+    )
+    X = data[0]
+    y = data[1]
+    X_train, X_test, y_train, y_true = train_test_split(
+        X, y, test_size=0.2, random_state=0
+    )
```

Comment:
This code repeats itself, might be useful to export to a shared fixture or even better parameterize the model (the only line that's different).